src/App.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import store from './store';
+import Home from './components/Home/Home';
+import Navbar from './components/Navbar/Navbar';
+import Feature from './components/Feature/Feature';
+import LoginForm from './components/Login/LoginForm';
+import SignupForm from './components/Signup/Signup';
+import MyReservations from './components/MyReservations/MyReservations';
+import ReservationForm from './containers/reservationform/reservationform';
+import ProtectedRoute from './components/ProtectedRoute';
+import Footer from './components/Footer/Footer';
+import Dashboard from './components/Dashboard/Dashboard';
+import DoctorInfo from './components/Feature/DoctorInfo/DoctorInfo';
+import DoctorForm from './components/DoctorForm/DoctorForm';
+import './app.css';
+
+function App() {
```

Comment:
- [ ] I cannot login as any of the included admins, or even if I try to create a new one through the console:

![image](https://github.com/wolterbwambale/booking-appointment-capstone_frontend/assets/57421823/7c278358-c354-4294-956a-3f26baecd5b1)

![image](https://github.com/wolterbwambale/booking-appointment-capstone_frontend/assets/57421823/d5888503-f1b6-48a6-9943-8aa8d92e8ebc)

Kindly, make sure admins can login, so we can test the add and delete functionality on your project. 👍

Reply:
the log in is a user name as per requirements and password not email and password.